tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -229,17 +257,17 @@
     <value>Show hidden and system files</value>
     <comment>The header for the show hidden and system files setting.</comment>
   </data>
-  <data name="DevDriveOptimizationSuggestion" xml:space="preserve">
-    <value>Suggestion</value>
-    <comment>Dev drive optimization suggestion</comment>
+  <data name="UntrustedDevDriveDescription" xml:space="preserve">
+    <value>Mircosoft Defender performance mode is only available for Trusted Dev Drive volumes </value>
+    <comment>The meaning of untrusted dev drives.</comment>
   </data>
-  <data name="DevDriveOptimized" xml:space="preserve">
-    <value>Optimized</value>
-    <comment>Dev drive optimized indicator</comment>
+  <data name="UntrustedDevDriveLearnMoreText" xml:space="preserve">
+    <value>Learn more.</value>
```

Comment:
"Learn more" links don't usually have a period
```suggestion
    <value>Learn more</value>
```

Reply:
Figma had period. Deleting it.